cds_dojson/marc21/fields/books/multipart.py

```diff
@@ -173,8 +285,77 @@ def number_of_volumes(self, key, value):
     raise IgnoreKey('number_of_volumes')
 
 
-@model.over('book_series', '^490__')
+@model.over('book_series', '^490__', override=True)
```

Comment:
why did you modify the rule for 490 ? It is serial title, nothing related to multipart monographs, did something change?

Reply:
I think you're right, this was a confusion on my side.